<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>826</width>
    <height>456</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Capturador de partículas</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Agregar</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Partículas</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="4" rowspan="15">
            <widget class="QPlainTextEdit" name="plainTextEdit"/>
           </item>
           <item row="4" column="3">
            <widget class="QSpinBox" name="spinBox_11">
             <property name="maximum">
              <number>500</number>
             </property>
            </widget>
           </item>
           <item row="1" column="3">
            <widget class="QSpinBox" name="spinBox_13">
             <property name="maximum">
              <number>500</number>
             </property>
            </widget>
           </item>
           <item row="6" column="0" colspan="2">
            <widget class="QLabel" name="label_26">
             <property name="text">
              <string>Color (rgb):</string>
             </property>
            </widget>
           </item>
           <item row="2" column="3">
            <widget class="QSpinBox" name="spinBox_16">
             <property name="maximum">
              <number>500</number>
             </property>
            </widget>
           </item>
           <item row="8" column="1">
            <widget class="QLabel" name="label_24">
             <property name="text">
              <string>Green</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0" colspan="2">
            <widget class="QLabel" name="label_27">
             <property name="text">
              <string>Destino en x:</string>
             </property>
            </widget>
           </item>
           <item row="7" column="1">
            <widget class="QLabel" name="label_22">
             <property name="text">
              <string>Red</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_23">
             <property name="text">
              <string>ID:</string>
             </property>
            </widget>
           </item>
           <item row="5" column="3">
            <widget class="QLineEdit" name="lineEdit"/>
           </item>
           <item row="8" column="3">
            <widget class="QSpinBox" name="spinBox_10">
             <property name="maximum">
              <number>255</number>
             </property>
            </widget>
           </item>
           <item row="2" column="0" colspan="2">
            <widget class="QLabel" name="label_29">
             <property name="text">
              <string>Origen en y:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0" colspan="2">
            <widget class="QLabel" name="label_21">
             <property name="text">
              <string>Origen en x:</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0" colspan="2">
            <widget class="QLabel" name="label_30">
             <property name="text">
              <string>Velocidad:</string>
             </property>
            </widget>
           </item>
           <item row="9" column="3">
            <widget class="QSpinBox" name="spinBox_14">
             <property name="maximum">
              <number>255</number>
             </property>
            </widget>
           </item>
           <item row="4" column="0" colspan="2">
            <widget class="QLabel" name="label_25">
             <property name="text">
              <string>Destino en y:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="3">
            <widget class="QSpinBox" name="spinBox_15">
             <property name="maximum">
              <number>500</number>
             </property>
            </widget>
           </item>
           <item row="0" column="1" colspan="2">
            <widget class="QLineEdit" name="lineEdit_2"/>
           </item>
           <item row="9" column="1">
            <widget class="QLabel" name="label_28">
             <property name="text">
              <string>Blue</string>
             </property>
            </widget>
           </item>
           <item row="7" column="3">
            <widget class="QSpinBox" name="spinBox_9">
             <property name="maximum">
              <number>255</number>
             </property>
            </widget>
           </item>
           <item row="11" column="3">
            <widget class="QPushButton" name="ordenar_id_pushButton">
             <property name="text">
              <string>Ordenar por ID</string>
             </property>
            </widget>
           </item>
           <item row="12" column="3">
            <widget class="QPushButton" name="ordenar_distancia_pushButton">
             <property name="text">
              <string>Ordenar por Distancia</string>
             </property>
            </widget>
           </item>
           <item row="13" column="3">
            <widget class="QPushButton" name="ordenar_velocidad_pushButton">
             <property name="text">
              <string>Ordenar por Velocidad</string>
             </property>
            </widget>
           </item>
           <item row="11" column="2">
            <widget class="QPushButton" name="pushButton">
             <property name="text">
              <string>Agregar al final</string>
             </property>
            </widget>
           </item>
           <item row="11" column="1">
            <widget class="QPushButton" name="pushButton_3">
             <property name="text">
              <string>Agregar al inicio</string>
             </property>
            </widget>
           </item>
           <item row="13" column="1" colspan="2">
            <widget class="QPushButton" name="pushButton_4">
             <property name="text">
              <string>Mostrar</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Tabla</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0" colspan="4">
         <widget class="QTableWidget" name="tabla"/>
        </item>
        <item row="1" column="3">
         <widget class="QPushButton" name="mostrar_tabla_pushButton">
          <property name="text">
           <string>Mostrar </string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QLineEdit" name="buscar_lineEdit"/>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="buscar_pushButton">
          <property name="text">
           <string>Buscar</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Dibujar</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0" colspan="2">
         <widget class="QGraphicsView" name="graphicsView"/>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="dibujar">
          <property name="text">
           <string>Dibujar</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="limpiar">
          <property name="text">
           <string>Limpiar</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbrir">
   <property name="text">
    <string>Abrir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionGuardar">
   <property name="text">
    <string>Guardar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
